streamly.cabal

```diff
@@ -102,6 +102,7 @@ extra-source-files:
     test/Streamly/Test/Data/Array/Foreign.hs
     test/Streamly/Test/Data/Array/Stream/Foreign.hs
     test/Streamly/Test/Data/Parser/ParserD.hs
+    test/Streamly/Test/Data/Stream/Top.hs
```

Comment:
We already have a `test/Streamly/Test/Prelude/Top.hs` module, why not add these tests in that module?